capture-video-frame.js

```diff
@@ -5,12 +5,13 @@
         root.captureVideoFrame = factory();
     }
 }(this, function () {
-    return function captureVideoFrame(video, format) {
+    return function captureVideoFrame(video, format, quality) {
         if (typeof video === 'string') {
             video = document.getElementById(video);
         }
 
         format = format || 'jpeg';
+        quality = quality || 1;
```

Comment:
MDN says `0.92` is the default, can you use that?

Reply:
oh, yep, I've just noticed that too.